scripts/lib/auth.js

```diff
@@ -167,9 +196,11 @@ function login(username, password) {
 /**
  * Log out.
  *
+ * @param {function} cb Callback to fire once stored auth credentials have been
+ *                      cleared. This is primarily for testing.
  * @return {Promise}
  */
-function logout() {
+function logout(cb) {
```

Comment:
I see that the callback is used in tests, but why can't the test use promises instead of a callback?